Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -239,6 +239,25 @@ public float CurrentLocomotiveSteamHeatBoilerWaterCapacityL
         //float DebugSpeed = 5.0f; // Initialise at 5 mph
         //float DebugTimer = 0.0f;
 
+        public enum SteamLocomotiveFuelTypes
+        {
+            Unknown,
+            Oil,
+            Wood, // not used at the moment
```

Comment:
```suggestion
            Wood,
```
I'm guessing this comment no longer applies.

Reply:
Ok, thanks for identifying this.